util/vec-map/src/lib.rs

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Provides a map (key-value store) interface backed by `heapless::Vec`
+
+#![deny(missing_docs)]
```

Comment:
I think this should have the `no_std` attribute

Reply:
+1
```suggestion
#![no_std]
#![deny(missing_docs)]
```